tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,18 +123,467 @@ def test_rgw_obc_quota(
+                marks=[
+                    pytest.mark.polarion_id("OCS-8081"),
+                ],
+            ),
+        ],
+    )
+    def test_rgw_obc_size_quota(
+        self,
+        awscli_pod_session,
+        rgw_bucket_factory,
+        test_directory_setup,
+        mcg_obj_session,
```

Ruff (0.16.0)

[warning] Unused method argument: `mcg_obj_session`

(ARG002)